docs/content/any/getting-started/quickstart/index.md

```diff
@@ -1,192 +1,246 @@
 ---
 title: Quickstart (5 min)
 description: |
-  Ready to get started? See Oso in action, and walk through our quick
-  tutorial for adding authorization to a simple web server.
+  Ready to get started? See Oso in action, and walk through adding roles to an app.
 weight: 1
 ---
 
-# Oso in 5 minutes
+# Oso Quickstart
 
-Oso helps developers build authorization into their applications. If you’ve
-never used Oso before and want to see it in action, this guide is for you.
-We’re going to walk through how to use Oso to add authorization to a simple web
-server.
+Oso is an open-source, batteries-included library for authorizing actions in your app.
+Out of the box, Oso lets you give your users roles and lets you specify permissions for those roles.
+Roles can be as simple as User and Admin, or as complex as a management hierarchy.
 
-{{% callout "Try it!" "green" %}}
-  To follow along, clone the {{% exampleGet "githubApp" %}}:
+[DIAGRAM]
 
-  ```console
-  git clone {{% exampleGet "githubURL" %}}
-  ```
-{{% /callout %}}
-
-## Run the server
+Oso isn’t restricted to roles, though — you can replace any authorization code in your app with an Oso policy.
 
-Our sample application serves data about expenses submitted by users. The
-sample application has three important files.
+Why use Oso?
```

Comment:
Just copying @litonico's response from the Google Doc here:

> Annie and I would like to keep this, or some version of this — we think it's important to tell people very clearly why Oso would be a good usage fit for them. We thought these statements were factual, and were good reasons for someone to use Oso. Are there different statements along these lines that you'd suggest?

- Authorization always starts out simple, but can be increasingly difficult to manage as your app grows.
- Authorization is security and should be as reliable as possible.
- If you’re not an authorization expert, Oso’s design will guide you to best practices.
 
---

- Oso provides a consistent model—an authorization policy—for expressing authorization logic, helping you manage it as your app grows.
- Oso is built on a Rust core using security best practices so that your authorization is reliable and bug free.
- Oso's design will guide you towards authorization best practices, so that you can implement authorization models that are easy to understand and extend without needing to become an expert.

I don't feel great about the second point about security, but something along these lines. These points give some concise evidence, making them more grounded and factual. What do you think? The 3rd could still be a bit more concrete -- what is it about the design that guides you towards best practices? What are the best practices? It might be hard to say this in one sentence (but I think it would be good to keep this section brief).

One concise example of a best practice that we have in the product is deny-by-default semantics in Polar.

Reply:
I do like having a bit of additional sell in here, but trimmed it down a little. This isn't the time to change the product messaging to be about security.